Source/Dafny/DafnyPipeline.csproj

```diff
@@ -34,4 +34,19 @@
   <ItemGroup>
```

Comment:
Shouldn't `DafnyPipeline` be published as package for Dafny as a library use-cases?

Reply:
I think so, but I was thinking that we'd make the modifications you mention [here](https://github.com/dafny-lang/dafny/issues/2084#issuecomment-1115870493) before doing that. If you think the library would be useful as-is then, yeah, let's include it.

Reply:
I think we should publish them now. Customers can always copy things from DafnyDriver if they're missing something that's only there.